site/en/docs/extensions/mv3/promises/index.md

```diff
@@ -55,12 +55,14 @@ Not all methods in extensions APIs support promises. Sometimes that's because we
 promise support on the method yet; in many cases it's because using a promise isn't feasible for the
 method.
 
-{% if false %}
 You can check if an API method support promises by checking its API reference page:
 
-{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/lDjGweNENKyqGSA2nZrp.png", alt="Screenshot of an API
-reference page with promises support noted", width="800", height="467" %}
-{% endif %}
+{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/3gwYoRgPRWOrs5N2j5mP.png", alt="Screenshot showing a
+method that supports promises", width="800", height="280" %}
```

Comment:
At the moment it's a bit hard to see what's going on in this page when viewed on desktop. On first glance it's hard to tell if the image is a screenshot of another page, or if it's an embedded block fo text that has slightly different rendering. At mobile sizes it's more obvious because the image scaling makes it more obvious that it's not text, but it still doesn't read to me as a screenshot.

I think we could address this by adding some UI treatment to the screenshot. A low effort solution would be to add, say, a 1px solid black outline to the image to more clearly offset it from the page's text content. A higher effort approach would be to capture a screenshot of a Chrome window that's sized to show off the specific content we want to highlight on an API doc page.

I just realized that even after looking at this image for a while and writing up this feedback, I still thought the "Methods" header in the screenshot was part of the page's text content. 🤦

Reply:
Done.